### ISSUE TITLE
Tasks are missing from the documentation.

### ISSUE THREAD

**Issue by zStupan** [NiaOrg/NiaPy#315]
All the task classes are missing from the API docs.

**Comment by GregaVrbancic** [comment]
@zStupan true, I don't really know why this happened. It seems that everything is configured correctly.

**Comment by zStupan** [comment]
I'm guessing it's because the classes aren't imported in niapy/__init__.py. It generates the docs for the runner class because it's imported as `from niapy.runner import Runner` and added to __all__.

Adding
```rst
.. automodule:: niapy.task
    :members:
    :undoc-members:
    :show-inheritance:

```
to docs/source/api/niapy.rst works.

But I think it would be better if runner and task were subsections of niapy like so:
```rst
:mod:`niapy`
============

:mod:`niapy.runner`
-------------------------
.. automodule:: niapy.runner
    :members:
    :undoc-members:
    :show-inheritance:

:mod:`niapy.task`
----------------------
.. automodule:: niapy.task
    :members:
    :undoc-members:
    :show-inheritance:
```

Then, the api docs would look like this:
- niapy
  - niapy.runner
  - niapy.task
- niapy.algorithms
  - niapy.algorithms.basic
  - niapy.algorithms.modified
  - niapy.algorithms.other
- niapy.problems
- niapy.util

The util docs should also be split into subsections of argparser, array, distances, factory,...

**Comment by zStupan** [comment]
Fixed in #321

**Comment by GregaVrbancic** [comment]
Wow, awesome @zStupan! Thank you
